fvm/tests/default_kernel/util.rs

```diff
@@ -0,0 +1,16 @@
+use std::rc::Rc;
```

Comment:
I will be adding more util functions (and the macro for checking errors) later so I would like to leave it, but it does seem like overkill at the moment

Reply:
Up to you. My general policy is to _not_ plan for the future like that, because the future rarely turns out as expected.

Also note: `util` crates/modules tend to cause more problems than they solve in my experience.

- At first, put helpers next to the code in question.
- Later, move helpers to some more general location when it becomes clear where else you'll need them. Usually, at that point, it'll be pretty clear.

If you just stick them in a util module, the util module will just grow and grow and grow with random helpers.